docs/ocis/adr/0028-activities-service.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "28. Activity Service"
+date: 2024-05-16T15:00:00+01:00
+weight: 28
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
+geekdocFilePath: 0028-activities-service.md
+---
+
+* Status: draft
+* Deciders: [@kobergj](https://github.com/kobergj), [@fschade](https://github.com/fschade)
+* Date: 2024-05-16
+
+## Context and Problem Statement
+
+The user should be able to see all activities for a resource.
+Besides the current resource, the user should also be able to decide if he wants to include child resource activities or not.
+
+## Decision Drivers <!-- optional -->
+
+* The user should be able to see all activities for a resource.
+* The user should be able to decide if he wants to include child resource activities.
+* Activities should be stored space efficiently.
+* Activities should be stored in a way that they can be queried efficiently.
+* Activities should stay in place event if the resource is gone.
+* Activities reflect the state at a given point in time and not the current state.
+* The Service should only store a configurable number of activities per resource.
+
+## Considered Options
+
+### Activity store
+
+* Use a go-micro store to store the individual activities.
```

Comment:
Can you stick to the ADR template and keep one headline per option?

Mabe make two ADRs one for store and one for format?

Reply:
can u explain what you mean, its a bit unclear to us?

Reply:
Or maybe even three ADRs: one for store, one for format, one for API.